templates/server/src/main/resources/archetype-resources/pom.xml

```diff
@@ -83,6 +84,9 @@
 
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <excludedGroups>${devonfw.test.excluded.groups}</excludedGroups>
```

Comment:
nice. So we also keep downward compatibility. Just to double check: Did you test this manually if it is working (e.g. to disable all but module tests with `-Pmoduletest`)?